src/modules/md/md.cpp

```diff
@@ -48,6 +49,17 @@ MDModule::MDModule() : Module("MD")
         "Only forces arising from intramolecular terms (including pair potential contributions) will be calculated",
         intramolecularForcesOnly_);
 
+    // Output
+    keywords_.add<OptionalIntegerKeyword>("Output", "EnergyFrequency",
+                                          "Frequency at which to calculate total system energy (or 0 to inhibit)",
+                                          energyFrequency_, 0, std::nullopt, 10, "Inhibited");
+    keywords_.add<OptionalIntegerKeyword>("Output", "OutputFrequency",
+                                          "Frequency at which to output step information (or 0 to inhibit)", outputFrequency_,
+                                          0, std::nullopt, 5, "Inhibited");
```

Comment:
For simplicity and consistency I would just go with "Off" for the special case here and in the others:
```suggestion
                                          0, std::nullopt, 5, "Off");
```